modules/cnv-understanding-live-migration.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-live-migration.adoc
+
+[id="cnv-understanding-live-migration_{context}"]
+= Understanding live migration
+
+Live migration is the process of moving a running virtual machine instance to 
+another node in the cluster without interruption to the virtual workload or 
+access. This can be a manual process, if you select a virtual machine instance 
+to migrate to another node, or an automatic process, if the 
+virtual machine instance has a `LiveMigrate` eviction strategy and the node on 
+which it is running is placed into maintenance. 
+
+[NOTE]
+====
+CNV 2.0 currently only supports live migration of virtual machines with 
```

Comment:
Not sure if it is relevant, but we also support migration of some non-shared types of disks:
containerDisks, secrets, configMaps, etc

Reply:
Hmm, storage is not my strong suit. Would it be better to phrase it:
"NFS shared volumes are the only shared data volume type supported for live migration in CNV 2.0."
?

Reply:
This variant is better for me, thanks